.github/workflows/check-examples.yml

```diff
@@ -1,13 +1,28 @@
 name: check examples
 
 on:
-  schedule:
-    # 04:00 every weekday morning
-    - cron: '0 4 * * 1-5'
+  pull_request:
+    branches:
+      - master
```

Comment:
Why only for master and not for develop?

Reply:
There is no `develop` branch in this repo.

Reply:
I was confused but the repo, then this is fine!